.github/workflows/ci_cd.yml

```diff
@@ -0,0 +1,76 @@
+name: CI/CD
+
+on:
+  push:
+    branches: 
+    - master
+  pull_request:
+    branches: 
+    - master
+
+jobs:
+  test:
+    name: Test on node ${{ matrix.node_version }} and ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        node_version:
+        - 10
+        - 12
+        os: 
+        - ubuntu-latest
+        - windows-latest
+        - macOS-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Setting up Node.js ${{ matrix.node_version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node_version }}
+
+    - name: Get npm cache directory
+      id: npm-cache
+      run: |
+        echo "::set-output name=dir::$(npm config get cache)"
+    - uses: actions/cache@v1
+      with:
+        path: ${{ steps.npm-cache.outputs.dir }}
+        key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
+        restore-keys: |
+          ${{ runner.os }}-node-
+          
+    - name: Installing npm modules
+      run: npm install
+
+    - name: Running Tests
+      run: npm run test
+
+    - name: Electron Builder Action
+      uses: samuelmeuli/action-electron-builder@v1.3.0
+      with:
+        github_token: ${{ secrets.github_token }}
+        mac_certs: ${{ secrets.mac_certs }}
+        mac_certs_password: ${{ secrets.mac_certs_password }}
+
+    - name: See files in dist folder
+      run: ls ./dist
+
+    - name: Cleanup artifacts
+      if: matrix.os != 'windows-latest'
+      run: |
+          mkdir artifacts
+          mv "dist/(*.exe,*.deb,*.AppImage,*.dmg)" artifacts || true
+
+    - name: Cleanup artifacts Win
+      if: matrix.os == 'windows-latest'
+      run: |
+          mkdir artifacts
+          mv dist/*.exe artifacts
+
+    - name: Deploying to GPR
+      uses: actions/upload-artifact@v1.0.0
+      with:
+        name: Zulip - ${{matrix.os}} -${{matrix.node_version}}
```

Comment:
Drop the `${matrix.node_version}` since that is not useful. Also, can you post the screenshot of how it looks in github so we can figure out what exactly to name this?

Reply:
I saw how it looks in the GitHub UI. We should definitely not include the node version in its name. Likewise, `Zulip -` is also unnecessary. Just having an OS name will suffice here.

Reply:
If we enter the following:
```
- name: Uploading artifacts
      uses: actions/upload-artifact@v1.0.0
      with:
        name: ${{matrix.os}}
```
as output we get this 
![Screenshot](https://user-images.githubusercontent.com/25535045/77190605-92c7e500-6aff-11ea-92db-13a1abbca47e.png)

Reply:
Let's use `${{ runner.os }}` instead then. The current output is good too because we just need to know what platform an installer belongs to.